packages/adminrouter/extra/src/nginx.agent.conf

```diff
@@ -13,6 +13,6 @@ http {
         include includes/server/open/agent.conf;
 
         include /opt/mesosphere/etc/adminrouter-listen-open.conf;
-        include /opt/mesosphere/etc/adminrouter-tls.conf;
+        include /opt/mesosphere/etc/adminrouter-tls-agent.conf;
```

Comment:
This means that any manual changes made to `/opt/mesosphere/etc/adminrouter-tls.conf` will not apply.

@jgehrcke are we sure enough that this is a private-enough interface that we do not need to worry about this?

Reply:
Everything in /opt/mesosphere is private. In other words:
- `/opt/mesosphere` should be considered immutable
- manual modifications in `/opt/mesosphere` are not supported